crd-docs/crd/kiali.io_kialis.yaml

```
@@ -58,7 +58,7 @@ spec:
                 type: string
 
               istio_namespace:
-                description: "The namespace where Istio is installed. If left empty, it is assumed to be the same namespace as where Kiali is installed (i.e. `deployment.namespace`)."
+                description: "The namespace where Istio is installed. If unset it is assumed to be the same namespace as where Kiali is installed (i.e. `deployment.namespace`). Set to `<none>` for an external Kiali deployment model, when no Istio is installed on the cluster."
```

But it could be explicitly set to empty string (`""`) (which is the default, and why "unset" works). So it isn't just when it is "unset", it is also when it is empty string:

```suggestion
                description: "The namespace where Istio is installed. If unset or set to empty string (`""`) it is assumed to be the same namespace as where Kiali is installed (i.e. `deployment.namespace`). Set to `<none>` for an external Kiali deployment model, when no Istio is installed on the cluster."
```